docs/integrations/security-threat-detection/alert-logic.md

```diff
@@ -0,0 +1,64 @@
+---
+id: alert-logic
+title: Alert Logic
+sidebar_label: Alert Logic
+description: The Sumo Logic app for Alert Logic provides enhanced security operations by seamlessly combining the capabilities of Alert Logic's incident detection and response with Sumo Logic's advanced log management and analytics platform.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('/img/integrations/security-threat-detection/alert-logic-logo.png')} alt="thumbnail icon" width="100"/>
+
+The Sumo Logic app for Alert Logic provides enhanced security operations by seamlessly combining the capabilities of Alert Logic's incident detection and response with Sumo Logic's advanced log management and analytics platform. This powerful integration allows you to gain comprehensive insights into your security landscape, detect potential threats, and respond effectively to security incidents.
+
+Key features of the Alert Logic app include:
```

Comment:
@JV0812, you might want to make all the bullet items sentence case. (For example, change "Real-time Threat Visibility" to "Real-time threat visibility".) But it's not essential.